src/eventloop.jl

```diff
@@ -18,7 +18,15 @@ function eventloop(socket)
                 end
             finally
                 @async begin
-                    sleep(idle_delay[])
+                    if check_interact_loaded() && idle_delay[] < interact_min_delay
```

Comment:
The probably just
```jl
if isdefined(Main, :Interact) # eliminate this once Interact is updated
    sleep(0.25)
end
```
and get rid of `idle_delay[]`?  As far as I can tell, no packages are actually using `set_idle_delay`, so we can eliminate it.